db/migrate/20220822181605_create_post.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
+      t.integer :CommentCounter
+

```

Comment:
- [OPTIONAL] According to the `ERD`, likes counter is part of the post, it will be nice to add that to your migrations. Something like so: 

```
t.integer :likesCounter

```